core/server/utils/startup-check.js

```diff
@@ -5,6 +5,7 @@ var packages = require('../../../package.json'),
     mode = process.env.NODE_ENV === undefined ? 'development' : process.env.NODE_ENV,
     appRoot = path.resolve(__dirname, '../../../'),
     configFilePath = process.env.GHOST_CONFIG || path.join(appRoot, 'config.js'),
+    config = require('../config'),
```

Comment:
Due to how and where these are run, you can't include config here - it needs to be done in the same way as it is in the other checks, E.g. https://github.com/TryGhost/Ghost/pull/6499/files#diff-61698d6ade731ad65fbc2225c40fd34dR183